File: JavaBasic/117_largest_digit.java
// 117	117_largest_digit.java	Find the largest digit in a given number.
public class _117_largest_digit {
    public static void main(String[] args) {
        int num = 39271, largest = 0, temp = num;
        while (temp > 0) { int d = temp%10; if(d>largest) largest=d; temp/=10; }
        System.out.println("Number: " + num + " | Largest digit: " + largest);
    }
}

File: JSBASIC/117_largest_digit.js
// 117	117_largest_digit.js	Find the largest digit in a given number.

let num = 39271;
let largest = 0;
let temp = num;
while (temp > 0) {
    let digit = temp % 10;
    if (digit > largest) largest = digit;
    temp = Math.floor(temp / 10);
}
console.log("Number:", num);
console.log("Largest digit:", largest);
